components/builder-api-proxy/plan.sh

```diff
@@ -6,7 +6,7 @@ pkg_license=("Apache-2.0")
 pkg_deps=(core/nginx core/curl core/coreutils)
 pkg_build_deps=(core/node8 core/gcc core/git core/tar core/phantomjs core/python2 core/make)
 pkg_svc_user="root"
-pkg_svc_group="root"
+pkg_svc_run="nginx -c ${pkg_svc_config_path}/nginx.conf"
```

Comment:
I believe this *should* work, although it might be better to try `pkg_svc_run="nginx -c {{pkg.svc_config_path}}/nginx.conf"` while does delay this to runtime but lets the Supervisor do the work (ex: https://github.com/fnichol/rocketry/blob/master/habitat/plan.sh#L16). The only catch (at the moment) is to add a fix to `do_build_service()` [like this](https://github.com/fnichol/rocketry/blob/master/habitat/plan.sh#L51-L57) until we fix the build program to put the `run` hook under `hooks/` (so that `run` always gets template-rendered).

However, if the `run` hook with this change works, ship it. It's already better in my book!

Reply:
I was following the example of other plans that do something similar. I also compared the generated run hook to what we used to have and it was basically identical.

Reply:
Awesome, I wasn’t sure either way

Reply:
Awesome, I wasn’t sure either way